src/containers/Range/index.js

```diff
@@ -17,8 +17,8 @@ const mapStateToProps = (state, ownProps) => {
 
 const mapDispatchToProps = (dispatch, ownProps) => {
   return {
-    onSubmit: (value) => {
-      dispatch(nodeRangeUpdate(ownProps.nodeId, value))
+    onSubmit: (obj) => {
+      dispatch(nodeUpdate(ownProps.nodeId, obj))
```

Comment:
`nodeUpdate` is an easy way to change the properties of a node without having to create a custom action/reducer for it. Also, just note that all of your handling of this was in the macros section for some reason.